tools/lzld/lzld.m

```diff
@@ -0,0 +1,24 @@
+#import <dlfcn.h>
```

Comment:
Add more comments here - add as much context as possible. If someone ever needs to dive into this, we don't want them to need to spend a day of research time.